src/config.rs

```diff
@@ -199,21 +199,32 @@ pub enum Config {
     /// The primary email address. Also see `SecondaryAddrs`.
     ConfiguredAddr,
 
+    /// List of configured IMAP servers as a JSON array.
+    ConfiguredMailServers,
```

Comment:
What do you think about renaming `ConfiguredMailServers` to `ConfiguredImapServers` and `ConfiguredSendServers` to `ConfiguredSmtpServers`? I think these names describe better what the config actually is. Especially 'mail server' is not really accurate because a mail server usually contains both an IMAP and an SMTP server.